docs/source/basic.rst

```diff
@@ -116,6 +116,63 @@ notify their running by creating a .pid file.
 
 
 
+Timeouts
```

Comment:
> What kind of enhance do you mean? I can do it, but what exactly?
To better explain in docstring, what it's for, and how will it influence executors :)

> The same usage, but not the same behavior. 
You're sure? It's allways checking condition for if it's started and if not at the end of a timeout, it kills executor and raises exception.